src/components/RegistrationLandingPage.tsx

```diff
@@ -35,11 +24,12 @@ export const RegistrationLandingPage = () => {
     control,
     handleSubmit,
     formState: { errors },
-  } = useForm<FormValues>({ resolver: yupResolver(schema), defaultValues });
+  } = useForm<FormValues>({ resolver: yupResolver(registrationSchema), defaultValues });
 
   const onSubmit = (data: FormValues) => {
-    console.log(data);
-    setEmail('email');
+    if (data) {
+      setEmail('email');
```

Comment:
`onSubmit` to jest od obsługi danych w formularzu, tutaj nie zmieniasz stanu emaila :)
Email będzie się inaczej zmieniał - za pewne poprzez jakieś `useEffect` po wejściu na strone i pobraniu z url /czy coś podobnego. Robimy jeszcze tą kwestie na backendzie, jak zostanie ogarnięta to wróci się tutaj do tego :)

Reply:
To akuratnie zostało zrobione tylko przez to, że z jakiegoś powodu nie chciało mi puścić "console.log(data) (no console log warning), natomiast funkcja która bierze jako parametr data musi go gdzieś użyć, inaczej znowu błąd. Proponuję na razie zostawić to w takiej formie, a jak już będziemy tam mieli coś konkretnego do dodania, to po prostu zmienić.